Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/security/LoginController.java

```diff
@@ -13,14 +10,5 @@
 public class LoginController {
   /** Riceve le chiamate all'endpoint REST per l'autenticazione. */
   @GetMapping
-  public Map<String, String> login(HttpSession session, Authentication authentication) {
-    if(authentication != null) {
-      var authorities = authentication.getAuthorities();
-      var adminAuthority = ProdulyticsGrantedAuthority.ADMIN.getAuthority();
-      var admin = authorities.stream().anyMatch(auth -> auth.getAuthority().equals(adminAuthority));
-      return Map.of("accessToken", session.getId(), "admin", Boolean.toString(admin));
-    } else {
-      return Map.of();
-    }
-  }
+  public void login() {}
```

Comment:
Non ho capito... perché lo teniamo, ma vuoto?

Reply:
Originalmente era così, è solo in seguito che lo abbiamo cambiato facendogli ritornare dei valori.
Si potrebbe provare a vedere cosa succede se lo togliessimo però.

Reply:
Perché dici che se lo togliamo c'è il rischio che il front-end si rompa? Direi che sarebbe anche meglio, così vediamo subito se ci siamo dimenticati di cambiare qualcosa, invece magari di usare un endpoint che non fa nulla.

Reply:
Vedi https://github.com/DeltaXswe/documentazione-interna/pull/499#discussion_r900896576